pg_search/src/schema/mod.rs

```diff
@@ -727,10 +727,6 @@ impl SearchIndexSchema {
             });
         }
 
-        // hardcode the ctid field into the schema.  "ctid" is a reserved Postgres attribute name
-        // so we don't need to worry about name conflicts
-        builder.add_u64_field("ctid", tantivy::schema::INDEXED | tantivy::schema::FAST);
-
```

Comment:
see above -- the creation of the "ctid" field in our schema has moved so that it'll also always be known as searchable by the schema.